### PR TITLE
enable saml for RN app

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
This PR enables SAML to be used in RN by setting an action we get a response instead of a redirect, allowing the app to take control

### CODE REVIEW

File: api/user.go [hunk 1]
```diff
@@ -1451,7 +1451,14 @@ func completeSaml(c *Context, w http.ResponseWriter, r *http.Request) {
 			http.Redirect(w, r, c.GetSiteURL()+val, http.StatusFound)
 			return
 		}
-		http.Redirect(w, r, app.GetProtocol(r)+"://"+r.Host, http.StatusFound)
+
+		if action == "mobile" {
+			user.Sanitize(map[string]bool{})
+
+			w.Write([]byte(""))
```
Comment (jwilander) [inline api/user.go]: Are we purposefully writing a blank string?

Comment (enahum) [inline api/user.go]: yes!

Comment (jwilander) [inline api/user.go]: Ok, then what's with sanitizing the user?

Comment (enahum) [inline api/user.go]: Lol that one escape, will remove it

**Comment by enahum** [comment]
@jwilander ready

**Pull request merged**